### ISSUE TITLE
Update wiki with instructions for developing using Docker

### ISSUE THREAD

**Issue by bergjohan** [include-cpp/include-cpp.github.io#5]
The following works on Linux. We should probably verify it on Windows and macOS before adding it to the wiki:

Install Docker:
See https://www.docker.com/

Clone the repository:
$ git clone git@github.com:include-cpp/include-cpp.github.io.git

Make sure the Docker service is running.

Run the site:
$ cd include-cpp.github.io/
$ docker compose up 

You should see something like:
include-cppgithubio-web-1  |     Server address: http://0.0.0.0:4000
include-cppgithubio-web-1  |   Server running... press ctrl-c to stop.

Enter the server address in a browser to see the site running.

Update the site:
Add a title in index.md and save it.
Refresh the site in your browser. You should now see the new title.

**Comment by claremacrae** [comment]
I have just tested these instructions on an Apple Silicon Mac.

- They did indeed show this URL, which gave me a blank page:
    - http://0.0.0.0:4000/
- Poking through Docker I saw this URL:
    - http://localhost:4000

The only other thought I had about the instructions is that most people will need to fork the repo, and clone the fork...

**Comment by claremacrae** [comment]
This wiki page is mostly still correct then, and it even mentions opening localhost..

https://github.com/include-cpp/include-cpp.github.io/wiki/Developing-the-Site

**Comment by bergjohan** [comment]
Right, what’s missing is instructions on installing Docker and running the Docker service, and running docker compose up, instead of having to install and setup Jekyll on your machine.

**Comment by claremacrae** [comment]
I checked and I don't have permission to edit the Wiki page - so I hope the info above is useful and someone else can edit it....
